lib/rails/generators/interactor/organizer_generator.rb

```diff
@@ -7,24 +7,35 @@ module Generators
     class OrganizerGenerator < ActiveInteractor::Generators::NamedBase
       source_root File.expand_path('templates', __dir__)
       desc 'Generate an interactor organizer'
-      argument :interactors, type: :array, default: [], banner: 'name name'
+      argument :interactors, type: :array, default: [], banner: 'interactor interactor'
+
+      class_option :context_attributes, type: :array, default: [], banner: 'attribute attribute', desc: "Attributes for the context"
+      class_option :skip_context, type: :boolean, desc: "Whether or not to generate a context class"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Prefer single-quoted strings when you don't need string interpolation or special symbols.](https://app.codacy.com/manual/aaronmallen/activeinteractor/pullRequest?prid=4773174)